python/pyspark/sql/column.py

```diff
@@ -462,6 +499,18 @@ def withField(self, fieldName: str, col: "Column") -> "Column":
 
         .. versionadded:: 3.1.0
 
+        Parameters
+        ----------
+        fieldName : str
+            a literal value.
+            The result will only be true at a location if any field matches in the Column.
+
+        Returns
+        -------
+        :class:`Column`
+            Column representing whether each element of Column
+            which field added/replaced by fileName.
```

Comment:
nit: `fieldName`

Reply:
Thanks, my mistake, updated in [f379ec3](https://github.com/apache/spark/pull/37521/commits/f379ec368c2309e353bba97d86ede443abb800ce)